src/Symfony/Component/Notifier/Bridge/Smsapi/SmsapiTransport.php

```diff
@@ -64,14 +64,25 @@ public function setTest(bool $test): static
 
     public function __toString(): string
     {
-        $dsn = sprintf('smsapi://%s?from=%s', $this->getEndpoint(), $this->from);
+        $dsn = sprintf('smsapi://%s', $this->getEndpoint());
+        $params = [];
+
+        if ('' !== $this->from) {
+            $params['from'] = (string) $this->from;
```

Comment:
this cast is useless as it is already a string

Reply:
https://github.com/symfony/symfony/pull/49261/commits/f49f88052d893c59e87f08e585531fdc34819d1e